tests/unit/test_paginate.py

```diff
@@ -218,7 +224,8 @@ def test_build_full_result_with_single_key(self):
             "result_key": "Users",
             "limit_key": "MaxKeys",
         }
-        self.paginator = Paginator(self.method, self.paginate_config)
+        self.paginator = Paginator(
+            self.method, self.paginate_config, mock.Mock())
```

Comment:
Wow these are a lot of tests to change. Could we just have a helper method for creating paginators so we do not have to update each test every time for next time?